AVPlayerDemo/Common/PlayerViewController.swift

```diff
@@ -42,40 +43,47 @@ class PlayerViewController: UIViewController {
     // Setup AVPlayerViewController view
     avpController.view.frame = self.view.bounds
     self.view.addSubview(avpController.view)
+  }
+  
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    airplayManager = LMAirplayManager(delegate: self)
+  }
+  
+  override func viewDidDisappear(_ animated: Bool) {
+    super.viewDidDisappear(animated)
+    plugin?.stop()
+    plugin = nil
```

Comment:
@borisborgobello do you think if we should `unregisterAirplayNotification()` here?

To me, it feel like we could end up having multiple call to `registerAirplayNotification()`